Project.toml

```
@@ -7,10 +7,12 @@ version = "0.1.0"
 FFTW = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
 ImageBase = "c817782e-172a-44cc-b673-b171935fbb9e"
 ImageFiltering = "6a3955dd-da59-5b1f-98d4-e7296123deb5"
+OffsetArrays = "6fe1bfb0-de20-5000-8ca7-80f57d26f881"
 
 [compat]
 FFTW = "1"
 ImageBase = "0.1.2"
+OffsetArrays = "1.10.5"
```

Actually "1" should be fine. As we always assume users are in the latest patch version so that bug should just be ignored.

```suggestion
OffsetArrays = "1"
```